tests/test_errors.py

```diff
@@ -320,6 +320,16 @@ def f(input_value, info):
     ('bytes_too_short', 'Data should have at least 1 byte', {'min_length': 1}),
     ('bytes_too_long', 'Data should have at most 42 bytes', {'max_length': 42}),
     ('bytes_too_long', 'Data should have at most 1 byte', {'max_length': 1}),
+    (
+        'bytes_invalid_encoding',
+        'Data should be valid base64, base64 error',
+        {'encoding': 'base64', 'encoding_error': 'base64 error'},
+    ),
+    (
+        'bytes_invalid_encoding',
+        'Data should be valid hex, hex error',
+        {'encoding': 'hex', 'encoding_error': 'hex error'},
+    ),
```

Comment:
The including of the encoding and encoding error seems redundant, are there any cases where having both is more informative? If not, let's simplify to just one

Reply:
I used an overly simplistic test fake here. In practice I think it says something useful, like the offset of the invalid character in the base64 input (like in `test_json_bytes_base64_invalid` below). I've updated these messages too to make that clearer.